account_page.php

```diff
@@ -294,6 +273,48 @@
 	</div>
 </div>
 
+<?php
+$t_projects = user_get_assigned_projects( auth_get_current_user_id() );
+if( count( $t_projects ) > 0 ) {
+?>
+	<div class="space-10"></div>
+
+	<div class="widget-box widget-color-blue2">
+		<div class="widget-header widget-header-small">
+			<h4 class="widget-title lighter">
+				<i class="ace-icon fa fa-user"></i>
```

Comment:
Should we use another icon for this section? Maybe the same one we use in manage projects page: http://fontawesome.io/icon/puzzle-piece/